src/constants.lua

```diff
@@ -36,70 +42,122 @@ function constants.hoursToMs(hours)
 	return hours * constants.minutesToMs(60)
 end
 
+-- TOKEN SUPPLY
+constants.TOTAL_TOKEN_SUPPLY = constants.ARIOToMARIO(1000000000) -- 1 billion tokens
```

Comment:
I almost yearn for the exponentiation instead of all those 0s here! Stresses me out counting them 😅

Reply:
Sure, we can make it `10 ^ constants.denomination` here too?

Reply:
total supply is 10 ^9 - the protocol supply is 50M (5 * (10 ^ 6)). the denomination indicates the mARIO/ARIO